packages/table/src/layout-observer.js

```diff
@@ -46,7 +46,10 @@ export default {
         const name = col.getAttribute('name');
         const column = columnsMap[name];
         if (column) {
-          col.setAttribute('width', column.realWidth || column.width);
+          const realWidth = (column.minWidth !== undefined && column.minWidth > column.width)
+            ? column.minWidth
+            : column.width;
```

Comment:
It seems there is no need to check the existense of `column.minWidth`.
I think it's more concise to use Math.max